contracts/HATKlerosConnector.sol

```diff
@@ -0,0 +1,486 @@
+     *  @param _loserMultiplier Multiplier for calculation the appeal cost of the losing side.
+     *  @param _loserAppealPeriodMultiplier Multiplier for calculating the appeal period for the losing side.
+     */
+    constructor (
+        IArbitrator _klerosArbitrator,
+        bytes memory _arbitratorExtraData,
+        HATArbitrator _hatArbitrator,
+        IHATVault _vault,
+        string memory _metaEvidence,
+        uint256 _winnerMultiplier,
+        uint256 _loserMultiplier,
+        uint256 _loserAppealPeriodMultiplier
+    ) {
+        emit MetaEvidence(0, _metaEvidence);
+        
+        governor = msg.sender;
+        klerosArbitrator = _klerosArbitrator;
+        arbitratorExtraData = _arbitratorExtraData;
+        hatArbitrator = _hatArbitrator;
+        vault = _vault;
+        winnerMultiplier = _winnerMultiplier;
+        loserMultiplier = _loserMultiplier;
+        loserAppealPeriodMultiplier = _loserAppealPeriodMultiplier;
+    }
+
+    // ******************** //
+    // *    Governance    * //
+    // ******************** //
+
+    /** @dev Changes winnerMultiplier variable.
+     *  @param _winnerMultiplier The new winnerMultiplier value.
+     */
+    function changeWinnerMultiplier(uint256 _winnerMultiplier) external onlyGovernor {
+        winnerMultiplier = _winnerMultiplier;
+    }
+
+    /** @dev Changes loserMultiplier variable.
+     *  @param _loserMultiplier The new winnerMultiplier value.
+     */
+    function changeLoserMultiplier(uint256 _loserMultiplier) external onlyGovernor {
+        loserMultiplier = _loserMultiplier;
+    }
+
+    /** @dev Changes loserAppealPeriodMultiplier variable.
+     *  @param _loserAppealPeriodMultiplier The new loserAppealPeriodMultiplier value.
+     */
+    function changeLoserAppealPeriodMultiplier(uint256 _loserAppealPeriodMultiplier) external onlyGovernor {
+        loserAppealPeriodMultiplier = _loserAppealPeriodMultiplier;
+    }
+
+    /** @dev Update the meta evidence used for disputes.
+     *  @param _metaEvidence URI of the new meta evidence.
+     */
+    function changeMetaEvidence(string calldata _metaEvidence) external onlyGovernor {
+        metaEvidenceUpdates++;
+        emit MetaEvidence(metaEvidenceUpdates, _metaEvidence);
+    }
+
+    // ******************** //
+    // *    Challenges    * //
+    // ******************** //
+    
+    /** @dev Notify KlerosArbitrator that expert's committee decision was challenged. Can only be called by Hat arbitrator.
+     *  Requires the arbitration fees to be paid.
+     *  @param _claimId The Id of the active claim in Vault contract.
+     *  @param _evidence URI of the evidence to support the challenge.
+     *  Note that the validity of the claim should be checked by Hat arbitrator.
+     *  @return surplusFee The amount of fee that exceeds required arbitration cost.
+     */
+    function notifyArbitrator(bytes32 _claimId, string calldata _evidence) external payable returns (uint256 surplusFee) {
```

Comment:
It would be useful to have an interface for V1 and V2 connectors which has this function so we can use it in the arbitrator contract

Reply:
I'll add it